index.js

```diff
@@ -126,10 +127,22 @@ const create = (win, options) => {
 						text: props.srcURL
 					});
 				}
+			}),
+			lookUpSelection: decorateMenuItem({
+				id: 'lookUpWord',
```

Comment:
Ooops! Looks like I missed the menu item  `id` when I changed the name from `lookUpWord` to `lookUpSelection`.
Do you want to reopen this PR or shall I open a new one?
Could also fix the ordering inconsistency mentioned in https://github.com/sindresorhus/electron-context-menu/pull/73#discussion_r288947453.

Reply:
Fixed that and another thing in https://github.com/sindresorhus/electron-context-menu/commit/28662f35ea238577a318f479cffaaff81d49d6ff (Make sure you fix that in your other PR too)

I noticed it's also missing from https://github.com/sindresorhus/electron-context-menu/blob/28662f35ea238577a318f479cffaaff81d49d6ff/index.d.ts#L12

Reply:
Thanks, and good catch re the missing bit from the readme. I'll fix that in my other PR.

I'm not suite sure when the `Labels` interface is used. I'll add the appropriate label to it for my other PR, but I'm not quite sure what to add here since the label is dynamically generated! Maybe just "Look Up" or "Look Up Selection"?

Reply:
Okay, I do see where it's used now.

Hmm, actually I guess the other PR doesn't want an entry in `Labels` at all since we don't define the label for the Services menu in `electron-context-menu`; it is defined by Electron (or maybe macOS).

Reply:
> Maybe just "Look Up" or "Look Up Selection"?

`Look Up`

> actually I guess the other PR doesn't want an entry in Labels at all since we don't define the label for the Services menu in electron-context-menu; it is defined by Electron (or maybe macOS).

Yup, but it still needs to be documented why it's missing from `Labels` and why it doesn't need to be localized.

Reply:
> `Look Up`

Alright. I'd better also add a comment to the effect that the default label is dynamically generated, which a localized replacement won't be.

> Yup, but it still needs to be documented why it's missing from `Labels` and why it doesn't need to be localized.

Okay. Just a comment in the appropriate place in `index.d.ts` for the Services item? Or should I add a `services` entry with a default value of, say, `undefined`, and a comment that it is not used and explaining why?